app/src/test/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImagePresenterTest.java

```diff
@@ -2,35 +2,43 @@
 
 import android.graphics.Bitmap;
 
+import com.jakewharton.retrofit2.adapter.rxjava2.HttpException;
+
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
 import org.mockito.junit.MockitoJUnitRunner;
 
+import java.util.UUID;
+
+import io.intrepid.contest.rest.EntryResponse;
 import io.intrepid.contest.screens.entrysubmission.entryimage.EntryImageContract.Presenter;
 import io.intrepid.contest.screens.entrysubmission.entryimage.EntryImageContract.View;
 import io.intrepid.contest.testutils.BasePresenterTest;
-import io.intrepid.contest.testutils.TestPresenterConfiguration;
+import io.reactivex.Observable;
 
+import static io.reactivex.Observable.error;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.reset;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
 public class EntryImagePresenterTest extends BasePresenterTest<EntryImagePresenter> {
     @Mock
     View mockView;
 
     private Presenter presenter;
+    private Throwable throwable;
```

Comment:
There's once branch we're not following down. I'll let you figure out which one. But we can verify if the alternative happens by using the "never()" term in mockito.